docs/running.md

```diff
@@ -1,9 +1,14 @@
 # Running on Helmi
 
-To run interactive jobs on LUMI the [`srun`](https://slurm.schedmd.com/srun.html) command can be used
+To run interactive jobs on LUMI the [`srun`](https://slurm.schedmd.com/srun.html) command can be used. First we allocate resources for the job:
 
 ```bash
-srun --account=project_<id> -t 0:02:00 -c 1 -n 1 --partition q_industry python qiskit_flip.py
+salloc -n 1 -c 1 -t 00:15:00 --job-name=<job_name> --account=project_<id> --partition q_industry
```

Comment:
We should follow the syntax of the official LUMI docs https://docs.lumi-supercomputer.eu/runjobs/scheduled-jobs/slurm-quickstart/ and therefore, not use `salloc`.`srun` is fine